cmd/panfs.go

```diff
@@ -1182,7 +1188,13 @@ func (fs *PANFSObjects) putObject(ctx context.Context, bucket string, object str
 	// so that cleaning it up will be easy if the server goes down.
 	tempObj := mustGetUUID()
 
-	fsTmpObjPath := pathJoin(fs.fsPath, minioMetaTmpBucket, fs.fsUUID, tempObj)
+	var fsTmpObjPath string
+	// This is to handle bucket metadata which is still using .minio.sys
+	if bucket != minioMetaBucket {
+		fsTmpObjPath = pathJoin(bucketDir, panfsMetaDir, "tmp", fs.fsUUID, tempObj)
```

Comment:
Please, don't use the `"tmp"` literal here. Is that what `tmpDir` at line 58 has been added for?

Reply:
Done